File: CRUD Operations in Asp.Net Core 2 Razor Page with Dapper and Repository Pattern/[C#]-CRUD Operations in Asp.Net Core 2 Razor Page with Dapper and Repository Pattern/C#/RazorPagesExample/Entity/Product.cs
using System.ComponentModel.DataAnnotations;

namespace RazorPagesExample.Entity
{
    public class Product
    {
        [Key]
        [Display(Name = "Product Id")]
        public int Id { get; set; }

        [Required]
        [Display(Name="Product Name")]
        [StringLength(25, ErrorMessage ="Name should be 1 to 25 char in lenght")]
        public string Name { get; set; }

        [Required]
        [Display(Name = "Model")]
        [StringLength(50, ErrorMessage = "Name should be 1 to 50 char in lenght")]
        public string Model { get; set; }

        [Required]
        [Display(Name = "Price")]        
        public int Price { get; set; }
    }
}

File: ASP.NET 4.5 Web Forms Tutorial Series in Visual Basic/[VB]-ASP.NET 4.5 Web Forms Tutorial Series in Visual Basic/VB/WingTipToys/Models/Product.vb
Imports System.ComponentModel.DataAnnotations

Public Class Product

    <ScaffoldColumn(False)> _
    Public Property ProductID() As Integer

    <Required, StringLength(100), Display(Name:="Name")> _
    Public Property ProductName() As String

    <Required, StringLength(10000), Display(Name:="Product Description"), DataType(DataType.MultilineText)> _
    Public Property Description() As String

    Public Property ImagePath() As String

    <Display(Name:="Price")> _
    Public Property UnitPrice() As Nullable(Of Double)

    Public Property CategoryID() As Nullable(Of Integer)

    Public Overridable Property Category() As Category

End Class